### PR TITLE
Address all collections.abc DeprecationWarnings

### PR DESCRIPTION
**Pull request by sloria**
#215 wasn't complete--this addresses the same `DeprecationWarning` everywhere.

### CODE REVIEW

File: flex/_compat.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""Python 2/3 compatibility module."""
+try:
+    from collections.abc import Mapping, Sequence
```
Comment (blag) [inline flex/_compat.py]:
```suggestion
    from collections.abc import Mapping, Sequence  # noqa: F401
```

File: flex/_compat.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+except ImportError:  # Python 2
+    from collections import Mapping, Sequence
```
Comment (blag) [inline flex/_compat.py]:
```suggestion
    from collections import Mapping, Sequence  # noqa: F401
```

**Comment by sloria** [comment]
Done

**Comment by blag** [comment]
Released version 6.14.1 to PyPI and GitHub with these changes. Thanks for your contribution!

**Comment by blag** [comment]
Also, thank you for following the pull request guidelines regarding gifs of cute animals. It's a very important part of the software development process (and an excellent way to bribe us maintainers).

**Pull request merged**